src/Components/DapperScrollbars/Documentation/DapperScrollbars.stories.tsx

```diff
@@ -55,46 +52,81 @@ scrollbarStories.add(
           scrollTop={number('scrollTop', 0)}
```

Comment:
These knobs no longer work because this prop is only used once as the initial state.

Reply:
Fixed